plugins/generator-1.14.4/forge-1.14.4/procedures/spawn_gem.java.ftl

```diff
@@ -1,6 +1,9 @@
 <#include "mcitems.ftl">
 if(world instanceof World && !world.getWorld().isRemote) {
 	ItemEntity entityToSpawn=new ItemEntity(world.getWorld(), ${input$x}, ${input$y}, ${input$z}, ${mappedMCItemToItemStackCode(input$block, 1)});
-	entityToSpawn.setPickupDelay(10);
+	entityToSpawn.setPickupDelay(${input$pickUpDelay!10});

```

Comment:
You will need to cast this to int. One can put double in this input and the procedure will not compile.

Reply:
done!